NewSpring/ApollosProject
# Adds Bugsnag error monitoring

## DESCRIPTION

### What does this PR do, or why is it needed?

Sets up Bugsnag Error Handling

Credentials to log in and API key can be found by asking Vincent or @IsaacHardy 

### How do I test this PR?

1. Try to sign in with an invalid username/password.
 - You should see an error show up in the bugsnag client project 
2. Change a resolver so it `throws new Error('Some error')`
3. Query that resolver
 - you should see an error show up in bugsnag server project
---

> I am affirming this is my _best_ work ([Ecclesiastes 9:10](https://www.bible.com/bible/97/ECC.9.10.MSG))

## TODO


## REVIEW

**Manual QA**


**Code Review: Questions to consider**


---

> The purpose of PR Review is to _improve the quality of the software._
